tests/text/inputs.py

```diff
@@ -60,6 +60,30 @@
     ],
 }
 
+# Examples and expected values taken from:
+# https://github.com/Tiiiger/bert_score/blob/master/tests/test_scorer.py
+ARTICLES_INPUT = {
+    "preds": [
+        "28-year-old chef found dead in San Francisco mall",
+        "A 28-year-old chef who recently moved to San Francisco was "
+        "found dead in the staircase of a local shopping center.",
+        "The victim's brother said he cannot imagine anyone who \
+        would want to harm him,\"Finally, it went uphill again at "
+        'him."',
+    ],
+    "targets": [
+        "28-Year-Old Chef Found Dead at San Francisco Mall",
+        "A 28-year-old chef who had recently moved to San Francisco \
+        was found dead in the stairwell of a local mall this "
+        "week.",
+        "But the victim's brother says he can't think of anyone who would \
+        want to hurt him, saying, \"Things were finally "
+        'going well for him."',
+    ],
+}
+
 _inputs_error_rate_batch_size_1 = Input(**ERROR_RATES_BATCHES_1)
 
 _inputs_error_rate_batch_size_2 = Input(**ERROR_RATES_BATCHES_2)
+
+_inputs_multiple_sentences_multiple_reference = Input(**ARTICLES_INPUT)
```

Comment:
Actually, there's a single reference for a given hypothesis.

Reply:
Yes. Should I call it `_inputs_multiple_sentences_single_reference`?

Reply:
Possibly we can. Or maybe we can leave the references in the test file for now. We aim to adjust `BERTScore` in a way to handle multiple references #647 (similar updates as you made for `ROUGEScore`) so we can eventually use already defined `_inputs_multiple_references`.

Reply:
Right. For now, I'll keep it as is in the current PR. We can rename it when issue #647 is completed.

I additionally had concerns that we should standardize naming conventions for preds (in some places hypothesis) and targets (in some places references) in the entirety of NLG metrics.

Reply:
@SkafteNicki ^^

Reply:
IMO we should go with predictions and targets everywhere, since this is then more consistent with metrics in other domains.

Reply:
agree here :]